tests/pkg/fixtures/big_requests.go

```diff
@@ -0,0 +1,58 @@
+package fixtures
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+var appGQLTemplate = `# Code generated by Compass integration tests, DO NOT EDIT.
+mutation {
+			result: registerApplication(in: {
+		name: "wordpress",
+		providerName: "provider name",
+		description: "my first wordpress application",
+		labels: {group:["production","experimental"],scenarios:["DEFAULT"],},
+		healthCheckURL: "http://mywordpress.com/health",
+    bundles: [{
+		name: "foo"
+		description: "Foo bar"
+      apiDefinitions: [ {
+      name: "comments-v1",
+      description: "api for adding comments",
+      targetURL: "http://mywordpress.com/comments",
+      group: "comments",
+      spec:{
+      data: "{\"openapi\":\"%s"}",	
+      type: OPEN_API,
+      format: YAML,
+    },
+      version:{
+      value: "v1",
+      deprecated: true,
+      deprecatedSince: "v5",
+      forRemoval: false,
+    },
+    }]
+  }]
+	}) {
+		id
+  }
+}`
+
+func FixHTTPBigBodyPOSTRequest(url string, bodySize int) (*http.Request, error) {
```

Comment:
Use query instead.
This file can be completely removed if you use FixGetApplicationRequest function from pkg/fixtures and just send the large content as ID - as you currently do here.